.github/openshift/deploy.backend.yml

```diff
@@ -40,6 +40,21 @@ parameters:
   - name: ALLOWED_ORIGINS
     description: Sets all the allowed request origins
     required: true
+  - name: DATABASE_HOST
+    description: Where the database is hosted
+    required: true
+  - name: DATABASE_PORT
+    description: The port to be used for the connection to the database
+    required: true
+  - name: SERVICE_NAME
+    description: The datanase's name
+    required: true
+  - name: DATABASE_USER
+    description: User for the database connection
+    required: true
+  - name: DATABASE_PASSWORD
+    description: Password for the database connection
+    required: true
```

Comment:
I noticed we haven't updated `docker-compose.yml` file with new env vars. In case we go with docker-composer we'll need those variables.

Reply:
I looked up the Docker Compose documentation and didn't find options to use the host's network in the container or list DNS address from the command line, so they were incorporated in the compose file.